msocasecharge.yaml

```diff
@@ -0,0 +1,183 @@
+entityDefinitions:
+
+  person:
+    fqn: "nc.SubjectIdentification"
+    entitySetName: "MSOInmates"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "PCP"
+    name: "person"
+ 
+  jailstay:
+    fqn: "ol.jailstay"
+    entitySetName: "MSOJailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SYSID"
+    name: "jailstay"
+
+  charge:
+    fqn: "ol.offense"
+    entitySetName: "MSOCharge"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "CHARGE_PK"
+      ol.eventcount:
+        type: "ol.eventcount"
+        column: "CHARGE_ORDER"
+      ol.type:
+        type: "ol.type"
+        column: "CHARGE_TYPE_ID"
+      ol.description:
+        type: "ol.description"
+        column: "OFFENSE_DESCRIPTION"
+      ol.code:
+        type: "ol.code"
+        column: "OFFENSE_CODE"
+    name: "charge"
+
+  chargeevent:
+    fqn: "ol.chargeevent"
+    entitySetName: "MSOChargeevent"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "CHARGE_PK"
+      ol.datetimestart:
+        type: "ol.datetimestart"
+        column: "OFFENSE_DATE"
+      ol.notes:
+        type: "ol.notes"
+        column: "COMMENTS"
+    name: "chargeevent"
+
+  disposition:
+    fqn: "ol.disposition"
+    entitySetName: "MSODisposition"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "DISPOSITION"
+      criminaljustice.disposition:
+        type: "criminaljustice.disposition"
+        column: "DISPOSITION"
+    name: "disposition"
+
+  bond:
+    fqn: "ol.bond"
+    entitySetName: "MSOBond"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "BOND_AMOUNT"
+      criminaljustice.bailbondamount:
+        type: "criminaljustice.bailbondamount"
+        column: "BOND_AMOUNT"
+    name: "bond"
+
+  case:
+    fqn: "justice.case"
+    entitySetName: "MSOCase"
+    propertyDefinitions:
+      j.CaseNumberText:
+        type: "j.CaseNumberText"
+        column: "CASE_PK"
+      justice.courtcasenumber:
+        type: "justice.courtcasenumber"
+        column: "CASE_PK"
+    name: "case"
+
+associationDefinitions:
+  relatedto:
+    fqn: "ol.relatedto"
+    entitySetName: "MSORelatedto"
+    src: "charge"
```

Comment:
I think this needs to be switched to charge event instead of charge

Reply:
done